jib-core/src/main/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGenerator.java

```diff
@@ -139,7 +139,7 @@ private static String mapToDockerfileString(Map<String, String> map, String comm
   @Nullable private String baseImage;
   private List<String> entrypoint = Collections.emptyList();
   private List<String> javaArguments = Collections.emptyList();
-  private Map<String, String> environment = Collections.emptyMap();
+  @Nullable private Map<String, String> environment;
```

Comment:
Should we have this be consistent with the others and not be nullable?

Reply:
entrypoint and javaArguments actually become nullable in a future change, I think.

Reply:
Ah right. I think if we make environment nullable, we should make exposedPorts and labels nullable too but that change can go in the post-release PR.